extensions/amp-story/1.0/amp-story-reaction-quiz.js

```diff
@@ -97,32 +96,22 @@ export class AmpStoryReactionQuiz extends AmpStoryReaction {
    * @param {Element} root
    */
   attachContent_(root) {
-    // TODO(jackbsteinberg): Optional prompt behavior must be implemented here
-    const promptInput = this.element.children[0];
     const promptContainer = root.querySelector(
       '.i-amphtml-story-reaction-quiz-prompt-container'
     );
 
-    // First child must be heading h1-h3
-    if (!['h1', 'h2', 'h3'].includes(promptInput.tagName.toLowerCase())) {
-      devAssert(
-        false,
-        'The first child must be a heading element <h1>, <h2>, or <h3>'
-      );
+    if (!this.element.hasAttribute('prompt-text')) {
       this.rootEl_.removeChild(promptContainer);
     } else {
-      const prompt = document.createElement(promptInput.tagName);
+      const prompt = document.createElement('h1');
```

Comment:
Some publishers will want to have full control over their Hx structure for SEO purposes. Can we use a `p` for now, and maybe switch to Hx titles if/when there's demand?